_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -41535,6 +41535,12 @@
 /obj/machinery/newscaster{
 	pixel_x = -30
 	},
+/obj/item/stack/sheet/mineral/plasma{
+	layer = 2.9
+	},
+/obj/item/stack/sheet/mineral/plasma{
+	layer = 2.9
```

Comment:
No, but it's present on all the others on the map too apparently. I'll add it to my list of things to clean up.